app/tests/test_consumer.py

```diff
@@ -50,17 +45,15 @@ def test_valid_message_writes_to_log_after_ftp(self, mock_deliver_binary, mock_s
             self.consumer.process(encrypted_jwt, uuid.uuid4())
         self.assertTrue(ConsumerTests._contains_statement_in_log_file("Delivered to FTP server", cm.output))
         self.assertTrue(mock_deliver_binary.called)
-        self.assertTrue(mock_send_receipt.called)
         self.assertTrue(mock_send_for_av_scan.called)
 
     @staticmethod
     def _contains_statement_in_log_file(statement, output):
         return [statement for line in output if statement in line]
 
     @patch('app.anti_virus_check.AntiVirusCheck.send_for_av_scan')
-    @patch('app.main.SeftConsumer._send_receipt')
     @patch('app.sdxftp.SDXFTP.deliver_binary')
-    def test_valid_message_receipt_sent(self, mock_deliver_binary, mock_send_receipt, mock_send_for_av_scan):
+    def test_valid_message_receipt_sent(self, mock_deliver_binary, mock_send_for_av_scan):
```

Comment:
Test name now misleading . Something along the lines of test_file_sent_to_av_and_ftp ?may be more appropriate ?

Reply:
Fixed